src/components/AssetManagementPanel/AssetManagementForm/AssetManagementDropdown/AssetManagementDropdown.tsx

```diff
@@ -7,12 +7,12 @@ import { AssetManagementActions } from "./../../AssetManagementActions";
 interface ManageAssetsDropdownProps {
   onSetFormAction: (nextFormAction: AssetManagementActions) => void;
   className?: string;
-  canSurrender: boolean;
+  canEndorseAndSurrender: boolean;
```

Comment:
why combine?

Reply:
ok u can decouple them but the checks are the same, we look for isBeneficiary and isHolder.